tests/unit/document/test_disable_offset2id.py

```diff
@@ -37,20 +42,6 @@ def test_offset2ids(docs, start_storage):
     list_like = False
     ids = [str(i) for i in range(3)]
     offset2id = Offset2ID(ids, list_like)
-    len_before = len(offset2id)
-    offset2id.append("4")
```

Comment:
why is this removed?

Reply:
Because the Offset2id class is not responsible for keeping the total length of Docarray list, it is no use to keep the functionality here.